keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -201,9 +202,11 @@ def augment_bounding_boxes(self, bounding_boxes, transformation, **kwargs):
         bboxes_out = tf.concat([min_cordinates, max_cordinates], axis=1)
         # cordinates cannot be float values, it is casted to int32
         bboxes_out = bounding_box.convert_format(
-            bboxes_out, source="xyxy", target=self.bounding_box_format
+            bboxes_out,
+            source="xyxy",
+            target=self.bounding_box_format,
```

Comment:
in keras we prefer full names to abbreviations: `bounding_boxes_out`

Let's update all of these to say `bounding_boxes` instead of bboxes

Reply:
Done